Resources/Prototypes/GameRules/roundstart.yml

```
@@ -244,7 +244,7 @@
   id: Nukeops
   components:
   - type: GameRule
-    minPlayers: 20
+    minPlayers: 50
```

25 would make sense for balancing with Cosmic Cult, 35 would probably be better. I'd prefer it closer to 30, but I'll side with @BombasterDS2 for 35 min, 40 max. Nukies should be able to reasonably roll on midpop.
```suggestion
    minPlayers: 30
```

---

35 would make it 4+1 (iirc 1 nukies per 10 crewmembers, 4 nukies + 1 agent). 40 is better. 

Still though - goob nukies are way too strong compared to crew, so it'll be stompfest anyways.